module_docstring.md

```diff
@@ -0,0 +1,33 @@
+# Changes made to autoDocstring to incorporate module level docstrings
```

Comment:
Thank you for this pr guide! Can you move it to a comment on the pr so that we can take it out if the code but ˆcan still read it?